evaluations/pose_graph_synthetic.sh

```diff
@@ -16,4 +16,4 @@ python examples/pose_graph/pose_graph_synthetic.py \
        dataset_size=32 \
        batch_size=32 \
        num_poses=4096 \
-       solver_type=baspacho
+       solver_type=lucuda
```

Comment:
hmm maybe cholmod is a better default since it works on both CPU and GPU?

Reply:
when solver_device=cpu then the solver_type is ignored. If I add baspacho as a cpu option it will not (currently) work because of the extra parameter dev="cpu" needed to the solver class, not supported by default! let's not forget this is an example script which was already there...